plc4j/api/src/main/java/org/apache/plc4x/java/api/model/PlcTag.java

```diff
@@ -63,6 +63,10 @@ default PlcValueType getPlcValueType() {
         return PlcValueType.NULL;
     }
 
+    @JsonIgnore
+    default void setPlcValueType(PlcValueType plcValueType){
+    }
```

Comment:
Hmm ... this adds a default implementation, which doesn't really do anything. Also do we usually implement all of the types in the API module, to create immutable types. Is there any reason for this addition?

Reply:
This seems to be related to the change in DefaultPlcWrite request, were the plc value type can be updated ... as I don't really like that change, I think we should also remove this change ... as both actually don't have anything to do with what we would like to address with this PR

Reply:
![image](https://user-images.githubusercontent.com/1769060/220872909-f6f07a09-6c5b-40ff-bd05-4e618a7fbd6f.png)

This is ready for PLC4X Driver. Maybe i should change these in next PR

Reply:
Yes please ... as I mentioned, we should try to keep one PR separated from the other ... we're already mixing up the Cache and the Timeouts (which I think is ok as I think both are not controversial)

Reply:
Ok. I will rollback